extra-kernel/libbpf/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=libbpf
+PKGSEC=kernel
+PKGDEP="elfutils"
+BUILDDEP="linux+api"
+BUILLDEP="rsync"
```

Comment:
Typo, and you were probably on the way to override your last `BUILDDEP` declaration.